CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## 0.2.0 - 2020-08-03
+
+### Added
+
+- Added `proxy_close_stream`.
+
+- Added `proxy_on_foreign_function`.
+
+### Changed
+
+- `proxy_on_request_headers` and `proxy_on_response_headers` added `end_of_stream`
+  flag as the 3rd argument.
+
+- `proxy_get_header_map_value` now returns `NOT_FOUND` instead of `OK` with
+  an empty value for non-existing keys.
+
+- Replaced `proxy_continue_request` with `proxy_continue_stream(HttpRequest)`.
+
+- Replaced `proxy_continue_response` with `proxy_continue_stream(HttpResponse)`.
+
+- Replaced `proxy_get_configuration` with `proxy_get_buffer_bytes(VmConfiguration)`
+  and `proxy_get_buffer_bytes(PluginConfiguration)`.
```

Comment:
nit: `VM_CONFIGURATION` and `PLUGIN_CONFIGURATION`

Reply:
Good catch, thanks!